app/commands/user/reputation_token/code_contribution/create.rb

```diff
@@ -0,0 +1,24 @@
+class User
+  class ReputationToken
+    module CodeContribution
+      class Create
+        include Mandate
+
+        initialize_with :user, :external_link, :repo, :number, :reason
```

Comment:
Tiny one, but I think I'd reorder the params so it's `:user, :reason, :repo, :pr_id, :external_link`

My logic is that these reputation token classes will always have a `user` and `reason` so having those first will give us some nice consistency. Then having the things that make up the `context_key` (repo and pr_id) feel good next, then finally an extra things.